realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy

```diff
@@ -174,11 +174,14 @@ class RealmTransformer extends Transform {
             it.getPackageName()
         }
 
+        def targetSdk = project?.android?.defaultConfig?.targetSdkVersion?.mApiLevel as String;
+        def minSdk = project?.android?.defaultConfig?.minSdkVersion?.mApiLevel as String;
```

Comment:
`targetSdkVersion` and `minSdkVersion` depend on product flavor. To collect more precise information,  we should obtain them from the product flavor currently building.

I'm looking for a way to determine the product flavor.